examples/gallery/src/mapbox-layer.html

```diff
@@ -11,11 +11,9 @@
 
     const {MapboxLayer, ScatterplotLayer} = deck;
 
-    mapboxgl.accessToken = '<mapbox-access-token>';
```

Comment:
This example uses Mapbox tilesets, so I think the token cannot be removed.

Reply:
Yes, you're right 👍  Fixed: e172c003c
BTW, I think that this example has not been working for a long time. ArcLayer was not defined, I've just imported it